lib/runner/cli/cli.js

```diff
@@ -11,6 +11,7 @@ const ProcessListener = require('../process-listener.js');
 const analyticsCollector = require('../../utils/analytics.js');
 const {Logger, singleSourceFile, isSafari, isLocalhost} = Utils;
 const {RealIosDeviceIdError, iosRealDeviceUDID, isRealIos, isMobile, killSimulator} = require('../../utils/mobile.js');
+const NightwatchEventHandler = require('../eventHandler.js');
```

Comment:
I would suggest to call this something like event publisher or event hub.

Reply:
Changed 👍